EnglishFilter/sections/foreign.txt

```diff
@@ -2392,6 +2392,8 @@ mob.hr##.side_banner
 ! https://github.com/AdguardTeam/AdguardFilters/issues/85200
 interez.sk###rklm-pod-clankom
 interez.sk##.sidebar > #custom_html-2
+! https://github.com/AdguardTeam/AdguardFilters/issues/86008
+sreality.cz###top-brand
```

Comment:
Ad is blocked by EasyList Czech and Slovak on my end. Please check again.

Reply:
@Alex-302, ad is visible on my side with Czech filter on.
<details><summary>Screenshot:</summary>

![screen](https://user-images.githubusercontent.com/68852737/123773276-36906b00-d8d5-11eb-8ab0-6692d6c715f9.jpg)
</details><br/>